plugins/modules/template_intent.py

```diff
@@ -277,7 +277,7 @@
         description: Description of template.
         type: str
       deviceTypes:
-        description: Configuration Template Create's deviceTypes.
+        description: Configuration Template Create's deviceTypes. This field is mandatory to create a new template.
```

Comment:
Can we change the description of devicetypes, rollbackTemplateParams and others

Reply:
Fixed.